build.gradle

```diff
@@ -464,6 +465,26 @@ project('spring-integration-test-support') {
     }
 }
 
+project('spring-integration-debezium') {
+    description = 'Spring Integration Debezium Support'
+    dependencies {
+        api project(':spring-integration-core')
+        api("io.debezium:debezium-api:$debeziumVersion") {
+            exclude group: 'org.slf4j'
+        }
+        api("io.debezium:debezium-embedded:$debeziumVersion") {
+            exclude group: 'org.slf4j', module: 'slf4j-reload4j'
+        }
+
+        testImplementation("io.debezium:debezium-connector-mysql:$debeziumVersion"){
+            exclude group: 'org.slf4j', module: 'slf4j-reload4j'
```

Comment:
Do we really see the problem with this `slf4j-reload4j`?
Why cannot we try to rely on a default state of dependency until we face a problem like in case of those duplicates mentioned above